src/Type/ObjectType.php

```diff
@@ -171,6 +171,21 @@ public function getIterableValueType(): Type
 		return new ErrorType();
 	}
 
+	public function isCallable(): TrinaryLogic
+	{
+		$broker = Broker::getInstance();
+
+		if (!$broker->hasClass($this->class)) {
+			return TrinaryLogic::createMaybe();
+		}
+
+		if ($broker->getClass($this->class)->hasMethod('__invoke')) {
+			return TrinaryLogic::createYes();
+		}
+
+		return TrinaryLogic::createNo();
```

Comment:
I think that `ObjectType(Closure::class)` should also be considered as yes.

Reply:
This already works because `Closure` has an `__invoke` method. But I've added test to make sure this continues to work in the future.

Reply:
I just checked the documentation of Closure and the method wasn't there, but I believe it can be reported in reflection. 👍